sqldelight-gradle-plugin/src/test/integration-migration-callbacks/src/test/java/com/squareup/sqldelight/integration/IntegrationTests.kt

```diff
@@ -0,0 +1,30 @@
+package com.squareup.sqldelight.integration
+
+import com.google.common.truth.Truth.assertThat
+import com.squareup.sqldelight.db.AfterVersion
+import com.squareup.sqldelight.db.migrateWithCallbacks
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver.Companion.IN_MEMORY
+import org.junit.Test
+
+class IntegrationTests {
+  @Test fun migrationCallbacks() {
+    val database = JdbcSqliteDriver(JdbcSqliteDriver.IN_MEMORY)
+    QueryWrapper.Schema.migrateWithCallbacks(
+      driver = database,
+      oldVersion = 0,
```

Comment:
Set this to 1 so we can test that the first callback is not invoked by dropping it from the test list below